js/src/layers/LocalTileLayer.js

```diff
@@ -0,0 +1,40 @@
+var widgets = require('@jupyter-widgets/base');
+var _ = require('underscore');
+var L = require('leaflet');
```

Comment:
Every time you have
```javascript
var L = require('leaflet');
```
It should be:
```javascript
var L = require('../leaflet.js');
```
Otherwise I think webpack will pick the `leaflet` library directly (without the plugins), not your file.